src/datamodel_code_generator/parser/jsonschema.py

```
@@ -1276,6 +1316,38 @@ class decorator which does not preserve staticmethod descriptors.
 
         return {}
 
+    def _get_python_type_base(self, python_type: str) -> str:  # noqa: PLR6301
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLR6301`)

Remove unused `noqa` directive

(RUF100)